jans-cedarling/bindings/cedarling_python/print_documentation.py

```
@@ -64,14 +64,18 @@ def print_doc(type_value: any, module_name: str | None = None):
+        return
+
     message = fix_newlines(type_value.__doc__)
     message_lines = message.split("\n")
-    if len(message_lines) != 0:
-        # check if in first line we have name of type
-        if not type_value.__name__ in message_lines[0]:
-            # add name of type to the first line
-            head_line = f"# {type_value.__name__}"
-            message_lines = [head_line] + message_lines
+    # check if in first line we have name of type
+    if not type_value.__name__ in message_lines[0]:
```

Ruff (0.15.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)